inventory/technical/reporting.rst

```diff
@@ -0,0 +1,191 @@
+==========================
+Inventory: Under the wood
+==========================
+
+The inventory valuation example
+********************************
+
+Let's take a look at the inventory valuation report.
+The report can be found in (Inventory -> Reporting -> Inventory Valuation)
+A popup appears asking if you want to compute the current inventory or the inventory at a specific date.
+
+inventory valuation of current inventory will redirect you to
+http://amazing_stock.odoo.com/web?#action=912&active_id=1&model=product.product&view_type=list&menu_id=
+
+inventory valuation of current inventory will redirect you to
+http://amazing_stock.odoo.com/web?#active_id=2&model=product.product&view_type=list&menu_id=
+
+Both are in fact different views on the model product.product.
+
+Let's set the context
+***********************
+The main differences are in the context:
+
+The context is a strange odoo concept: we use for communication between the client and the server.
```

Comment:
I would remove this :) 

I would say "the context is a multi-purpose dictionary passed alongside each call to Odoo"